VTUL/iawa
# Restyle to match university branding

**Restyle site to conform to university branding guidelines.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1762) (:star:)

# What does this Pull Request do? (:star:)
Restyle to match university branding guidelines for header and footer, etc.

# What's the changes? (:star:)
Restyle to match university branding guidelines for header and footer, etc.

# How should this be tested?

A description of what steps someone could take to:
* Check that header and footer match vt.edu on all pages.
* Check that nothing has been broken due to the merging of css rules and js code.

# Additional Notes:
* branch: `LIBTD-1762`

# Interested parties
@tingtingjh 

(:star:) Required fields
